docs/config/index.md

```diff
@@ -462,6 +462,13 @@ export default defineConfig(({ command, mode }) => {
   `envPrefix` should not be set as `''`, which will expose all your env variables and cause unexpected leaking of of sensitive information. Vite will throw error when detecting `''`.
   :::
 
+### spa
+
+- **Type:** `boolean`
+- **Default:** `true`
+
+  This configuration is meant for framework authors, or users using Vite's SSR API. See [SSR - Vite CLI](/guide/ssr#vite-cli).
```

Comment:
I don't know if we need to mention framework authors. We should probably define SPA. I'm not sure why linking to the CLI instead of SSR page generally. If we were going to keep the existing text, the comma is unnecessary
```suggestion
  Whether your application is a Single Page Application (SPA). Set to `false` if using Vite's [SSR](/guide/ssr) functionality.
```

Reply:
This sounds a bit strange. VitePress is going to use the SSR functionality but keep `spa: true`, no? Maybe something like this?
```
  Whether your application is a Single Page Application (SPA). Set to `false` for other kinds of apps like MPAs. Learn more in Vite's [SSR guide](/guide/ssr#vite-cli).
```

Reply:
>   Whether your application is a Single Page Application (SPA). Set to `false` for other kinds of apps like MPAs. Learn more in Vite's [SSR guide](/guide/ssr#vite-cli).

Committed: https://github.com/vitejs/vite/pull/8217/commits/71187f318e993949737f782ea27e9224fd60583c.